_opensearch/supported-field-types/xy-shape.md

```diff
@@ -0,0 +1,379 @@
+---
+layout: default
+title: xy shape
+nav_order: 59
+has_children: false
+parent: Cartesian field types
+grand_parent: Supported field types
+---
+
+# xy shape field type
+
+An xy shape field type contains a shape, such as a polygon or a collection of xy points. It is based on the Lucene [XYShape](https://lucene.apache.org/core/8_3_0/sandbox/org/apache/lucene/document/XYShape.html) field type. To index a geoshape, OpenSearch tesselates the shape into a triangular mesh and stores each triangle in a BKD tree. This provides a 10<sup>-7</sup>decimal degree of precision, which represents near-perfect spatial resolution.
```

Comment:
the lucene documentation link here is for 8.3.0 but above it is for 8.5.1 should we be consistent here?

Reply:
I think we can use lucene version that will be part of OpenSearch 2.4. wdyt @navneet1v ?

Reply:
I updated the link to 9.3.0